mobile-ui/mqttMiddleWare.js

```diff
@@ -0,0 +1,9 @@
+import {REACT_APP_AIO_USERNAME, REACT_APP_AIO_KEY} from '@env';
+import { publishDeviceStateAction } from './actions/device';
+export default mqttMiddleWare = (client) =>(store) => (next) => (action) => {
+    if (action.type === publishDeviceStateAction) {
+        console.log('publish', action.payload);
+        client.publish(action.payload.deviceId, action.payload.state)
```

Comment:
I may think publish function may cause error in case lost connection, and we need some solution to solve.
It can appear in another ways, but after user have clicked, and nothing change in user button, or delay in a little time